### PR TITLE
[Multi Hub] Tab Stage View Unfurling: NodeJS

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Hi @ChetanSharma-msft @Prithvi-MSFT

Raied PR for Tab Stage View META-OS Unfurling Updates (Nodejs)

Please review and provide feedback.

Thanks.

### CODE REVIEW

File: samples/tab-stage-view/nodejs/Readme.md [hunk 1]
```diff
@@ -163,6 +171,18 @@ Also update `BaseUrl` according to your code runtime environment.
 
 ![AppOffice](Images/AppOffice.png) 
 
+**After opening Outlook web, click the "New mail" button.**
+
+![Open New Mail](Images/OpenNewMail.png)
+
+**on the tool bar on top,select Apps icon. Your sideloaded app title appears among your installed apps**
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/nodejs/Readme.md:178]: **On the tool bar on top, select Apps icon. Your sideloaded app title appears among your installed apps**

File: samples/tab-stage-view/nodejs/server/bot/botActivityHandler.js [hunk 2]
```diff
@@ -21,6 +22,23 @@ class BotActivityHandler extends TeamsActivityHandler  {
           await next();
       });
     }
+
+    handleTeamsAppBasedLinkQuery(context, query) {
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/nodejs/server/bot/botActivityHandler.js:26]: Add method explanation or summary.

File: samples/tab-stage-view/nodejs/server/models/adaptiveCardUnfurling.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+// Adaptive Card for tab to show in stage view
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/nodejs/server/models/adaptiveCardUnfurling.js:1]: This adaptive care you are showing in tab or as a card for link unfurling?

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Kindly run build pipeline

File: samples/tab-stage-view/nodejs/Readme.md (hunk 1 above)
Comment (ChetanSharma-msft) [reply samples/tab-stage-view/nodejs/Readme.md:178]: @Mohammed-MSFT - This is not fixed

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Please execute build pipeline.

File: samples/tab-stage-view/nodejs/server/models/adaptiveCardUnfurling.js (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/tab-stage-view/nodejs/server/models/adaptiveCardUnfurling.js]: Its card as link unfurling which opens a tab in stage view

File: samples/tab-stage-view/nodejs/Readme.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-stage-view/nodejs/Readme.md]: added

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5345170590

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**